openstf/pipeline/create_forecast.py

```diff
@@ -72,13 +72,18 @@ def create_forecast_pipeline_core(
     validated_data = validation.validate(input_data)
 
     # Add features
-    data_with_features = OperationalPredictFeatureApplicator(
-        # TODO use saved feature_names (should be saved while training the model)
-        horizons=[0.25],
-        feature_names=model._Booster.feature_names,
-    ).add_features(validated_data)
+    data_with_features = (
+        OperationalPredictFeatureApplicator(
+            # TODO use saved feature_names (should be saved while training the model)
+            horizons=[0.25],
+            feature_names=model._Booster.feature_names,
+        )
+        .add_features(validated_data)
+        .drop(columns="load")
+    )
```

Comment:
Shouldnt 'data_with_features' also have the target-column (e.g. load)?

If not, perhaps another name for the variable would make more sense ('e.g. features or forecast_input or something else)

Reply:
yes good point, I now moved the dropping of the load column to the next step were "forecast_input_data" is created.